### PR TITLE
Mutational Signatures Methods

### PR DESCRIPTION
**Pull request by cansavvy**
#### Purpose

Adding mutational signature to the methods. Let me know if there is anything that needs further details added. 

I don't think the equation is completely necessary or actually clear how it is written, but we can remove it if it is just adding to confusion. 

#### Issue

#6 

#### Pull review checklist

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [x] All figures follow the [Manubot figure instructions](https://github.com/jaclyn-taroni/OpenPBTA-manuscript/blob/master/USAGE.md#figures)

### CODE REVIEW

File: content/03.methods.md [hunk 1]
```diff
@@ -168,6 +168,20 @@ We annotated putative driver fusions and prioritized fusions based on partners c
 _MYBL1_ [@doi:10.1073/pnas.1300252110], _SNCAIP_ [@doi:10.1038/nature11327], _FOXR2_ [@doi:10.1016/j.cell.2016.01.015], _TTYH1_ [@doi:10.1038/ng.2849], and _TERT_ [@doi:10.1038/ng.3438; @doi:10.1002/gcc.22110; @doi:10.1016/j.canlet.2014.11.057; @doi:10.1007/s11910-017-0722-5] were added to the oncogene list and _BCOR_ [@doi:10.1016/j.cell.2016.01.015] and _QKI_ [@doi:10.1038/ng.3500] were added to the tumor suppressor gene list based on pediatric cancer literature review.
 The fusion filtering workflow can be found in the [OpenPBTA Analysis repository](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion_filtering).
 
+### Mutational Signatures
+
+Using consensus SNVs, we obtained weights for signature sets using deconstructSigs [@doi:10.1186/s13059-016-0893-4 ; @url:https://github.com/raerose01/deconstructSigs] with BSgenome.Hsapiens.UCSC.hg38 annotation [@url:https://bioconductor.org/packages/release/data/annotation/html/BSgenome.Hsapiens.UCSC.hg38.html].
+We used each sample's signature weights to estimate how many mutations for each sample contributed to each signature.
```
Comment (cgreene) [inline content/03.methods.md]:
```suggestion
We estimated how many mutations contributed to each signature for each sample 
 using each sample's signature weights.
```

File: content/03.methods.md [hunk 2]
```diff
@@ -168,6 +168,20 @@ We annotated putative driver fusions and prioritized fusions based on partners c
 _MYBL1_ [@doi:10.1073/pnas.1300252110], _SNCAIP_ [@doi:10.1038/nature11327], _FOXR2_ [@doi:10.1016/j.cell.2016.01.015], _TTYH1_ [@doi:10.1038/ng.2849], and _TERT_ [@doi:10.1038/ng.3438; @doi:10.1002/gcc.22110; @doi:10.1016/j.canlet.2014.11.057; @doi:10.1007/s11910-017-0722-5] were added to the oncogene list and _BCOR_ [@doi:10.1016/j.cell.2016.01.015] and _QKI_ [@doi:10.1038/ng.3500] were added to the tumor suppressor gene list based on pediatric cancer literature review.
 The fusion filtering workflow can be found in the [OpenPBTA Analysis repository](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion_filtering).
 
+### Mutational Signatures
+
+Using consensus SNVs, we obtained weights for signature sets using deconstructSigs [@doi:10.1186/s13059-016-0893-4 ; @url:https://github.com/raerose01/deconstructSigs] with BSgenome.Hsapiens.UCSC.hg38 annotation [@url:https://bioconductor.org/packages/release/data/annotation/html/BSgenome.Hsapiens.UCSC.hg38.html].
```
Comment (cgreene) [inline content/03.methods.md]:
```suggestion
We obtained weights for signature sets by applying deconstructSigs [@doi:10.1186/s13059-016-0893-4 ; @url:https://github.com/raerose01/deconstructSigs] to consensus SNVs with the BSgenome.Hsapiens.UCSC.hg38 annotations [@url:https://bioconductor.org/packages/release/data/annotation/html/BSgenome.Hsapiens.UCSC.hg38.html].
```

File: content/03.methods.md [hunk 3]
```diff
@@ -168,6 +168,20 @@ We annotated putative driver fusions and prioritized fusions based on partners c
...
+We used each sample's signature weights to estimate how many mutations for each sample contributed to each signature.
+Weights are normalized from 0 to 1.
```
Comment (cgreene) [inline content/03.methods.md]:
```suggestion
We normalized weights from 0 to 1 with a linear transformation.
```

Comment (cgreene) [reply content/03.methods.md]: I think this is what you mean. Unless you are saying that the property of the weights is that they are distributed from zero to one.

File: content/03.methods.md [hunk 4]
```diff
@@ -168,6 +168,20 @@ We annotated putative driver fusions and prioritized fusions based on partners c
...
+
+For a given sample and signature:
```
Comment (cgreene) [inline content/03.methods.md]: What is this calculating? I think you could clarify by adding a bit more to this sentence fragment.

File: content/03.methods.md [hunk 5]
```diff
@@ -168,6 +168,20 @@ We annotated putative driver fusions and prioritized fusions based on partners c
...
+Using consensus SNVs, we obtained weights for signature sets using deconstructSigs [@doi:10.1186/s13059-016-0893-4 ; @url:https://github.com/raerose01/deconstructSigs] with BSgenome.Hsapiens.UCSC.hg38 annotation [@url:https://bioconductor.org/packages/release/data/annotation/html/BSgenome.Hsapiens.UCSC.hg38.html].
+We used each sample's signature weights to estimate how many mutations for each sample contributed to each signature.
+Weights are normalized from 0 to 1.
+
+For a given sample and signature:  
+
+$$ \frac{\textrm{# of contributing mutations}}{Mb} = \frac{\textrm{weight} * \sum{\textrm{# Trinucleotide mutations}}} {\textrm{Size in Mb of effectively surveyed genome}}
+$$
+
+Note that these reports do not take into account signatures of small contribution; deconstructSigs drops signature weights that are less than 6% [@doi:10.1186/s13059-016-0893-4].
```
Comment (cgreene) [inline content/03.methods.md]:
```suggestion
These results do not include signatures with small contributions; deconstructSigs drops signature weights that are less than 6% [@doi:10.1186/s13059-016-0893-4].
```

File: content/03.methods.md [hunk 6]
```diff
@@ -168,6 +168,20 @@ We annotated putative driver fusions and prioritized fusions based on partners c
...
+These methods were employed for both COSMIC [@url:https://cancer.sanger.ac.uk/cosmic] and Alexandrov et al, 2013 [@doi:10.1038/nature12477] signature sets.
```
Comment (cgreene) [inline content/03.methods.md]:
```suggestion
We used these methods to calculate signature scores for each sample with both COSMIC [@url:https://cancer.sanger.ac.uk/cosmic] and Alexandrov et al, 2013 [@doi:10.1038/nature12477] signature sets.
```

File: content/03.methods.md (hunk 3 above)
Comment (cansavvy) [reply content/03.methods.md]: The second thing you said. This is not a transformation I applied but I thing that is already a property of the weights obtained from deconstructSigs. But I felt understanding this helps explain the rationale for the number of contributing mutations estimation.

Comment (cgreene) [reply content/03.methods.md]: Ok - see if what I wrote seems ok.

**Pull request merged**